guides/P2PTunnels.md

```diff
@@ -0,0 +1,48 @@
+# How to Use P2P Tunnels in Applied Energistics 2
+AE2 (short for Applied Energistics 2) is a great mod for storing your items, autocrafting and much more. However once you'll get a big ME system going, you'll start running into some issues with **channels**.
+
+## Channels 
+Most of the things you can connect to your network (interfaces, ME Drives, ME chests, terminals, buses, crafting multiblocks etc.) use up a channel each, and channels are provided from your ME controller multiblock. These blocks also do transmit channels themselves. You can carry 8 channels with most cables, but dense cables allow you to carry 32.
+
+![2022-06-18_10 54 16](https://user-images.githubusercontent.com/100072467/174430656-b3ce42a5-1a8f-4c33-8601-edf96c5d9e14.png)
+We have 3 ME interfaces on the right, so they're getting 3 channels in total.
+
+Channels are provided to blocks from the shortest path possible, so make sure your cable lines aren't touching each other, since that might cause you some troubles in the long run.
+
+![2022-06-18_10 58 49](https://user-images.githubusercontent.com/100072467/174430682-33688f0f-b8f1-4f58-8872-d38801c06e62.png)
+
+**Colored pipes** don't connect to different colored pipes but do connect to fluix/uncolored pipes. **Cable anchors** prevent cables from connecting from a specific place.
+
+![2022-06-18_11 01 56](https://user-images.githubusercontent.com/100072467/174430704-fdee4dc1-19e4-4116-801c-1d286dd1a959.png)
+
+These will help you manage your cables to some extent. But there is something much more powerful that you can use.
+
+## The P2P Tunnel
+P2P (stands for point-to-point) tunnels act like a link between two points connected with some cable. They can carry a certain thing like items, fluids, energy and even light. But most importantly, they can carry channels.
+
+![2022-06-18_11 18 54](https://user-images.githubusercontent.com/100072467/174430735-4702ca6e-efb3-4847-9234-53a5ad52ea55.png)
+Here, we have a P2P tunnel carrying the light of the torch on the left side to right side.
+
+In order to function, they too need a channel each, but **they can't use the channels they get from the input side**, so you'll need to make sure there is a channel for each one given to them from the back. There is a misconception that they compress channels, but that's untrue. Instead, they carry channels with a channel cost.
```

Comment:
> Instead, they carry channels with a channel cost.

perhaps "Instead, they can carry up to 32 channels at a 2-channel cost."?

Reply:
Might be better to say "up to 32 channels at a 2-channel cost at minimum" since you can have multiple outputs.

Reply:
"each needing a channel" is probably my best bet then, also even with 2 P2Ps, two channels would be confusing anyway since you can seperate the paths and stuff